source/_patterns/02-molecules/vue-widgets/src/vue-clock/vue-clock.vue

```diff
@@ -0,0 +1,210 @@
+        </div>
+        <div
+          class="hoursAxis"
+          :style="[hoursPosition]"
+        >
+          <div class="hoursHand" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'VueClock',
+  data() {
+    return {
+      time: {
+        seconds: 0,
+        minutes: 0,
+        hours: 0,
+      },
+      secondsPosition: {
+        transform: 'rotate(0deg)',
+      },
+      minutesPosition: {
+        transform: 'rotate(0deg)',
+      },
+      hoursPosition: {
+        transform: 'rotate(0deg)',
+      },
+      initial: true,
+    };
+  },
+  computed: {
+    dynamicClasses() {
+      const classObject = {
+        success:
+          (this.time.seconds < 5 && this.time.seconds >= 0) ||
+          (this.time.seconds < 45 && this.time.seconds >= 40),
+        primary:
+          (this.time.seconds < 10 && this.time.seconds >= 5) ||
+          (this.time.seconds < 50 && this.time.seconds >= 45),
+        salmon: this.time.seconds < 15 && this.time.seconds >= 10,
+        purple: this.time.seconds < 20 && this.time.seconds >= 15,
+        orange: this.time.seconds < 25 && this.time.seconds >= 20,
+        yellow: this.time.seconds < 30 && this.time.seconds >= 25,
+        dark: this.time.seconds < 35 && this.time.seconds >= 30,
+        cyan: this.time.seconds < 40 && this.time.seconds >= 35,
+      };
+      return classObject;
+    },
+  },
+  created() {
+    this.updatePosition();
+  },
+  methods: {
+    /**
+     * Updates the seconds position
+     */
+    updatePosition() {
+      this.updateClock = setInterval(() => {
+        const time = new Date()
+          .toString()
+          .split(' ')[4]
+          .split(':');
+
+        const seconds = time[2];
+
+        const minutes = time[1];
+
+        const hours = time[0];
+
+        if (this.initial) {
+          this.updateMinutesPosition(minutes);
+          this.updateHoursPosition(hours);
+          this.initial = false;
+        }
+
+        this.updateSecondsPosition(seconds);
+        if (this.secondsPosition === '00') this.updateMinutesPosition(minutes);
+        if (this.hoursPosition === '00') this.updateHoursPosition(hours);
+      }, 1000);
+    },
+    updateSecondsPosition(seconds) {
+      this.$set(this.time, 'seconds', seconds);
+      const degrees = 6 * parseInt(seconds, 10);
+      this.secondsPosition = {
+        transform: `rotate(${degrees}deg)`,
+      };
+    },
+    updateMinutesPosition(minutes) {
+      this.$set(this.time, 'minutes', minutes);
+      const degrees = 6 * parseInt(minutes, 10);
+      this.minutesPosition = {
+        transform: `rotate(${degrees}deg)`,
+      };
+    },
+    updateHoursPosition(hours) {
+      this.$set(this.time, 'hours', hours);
+      const degrees = 30 * parseInt(hours, 10);
+      this.hoursPosition = {
+        transform: `rotate(${degrees}deg)`,
+      };
+    },
+  },
+};
+</script>
+
+<style lang="scss" scope>
```

Comment:
How about we use the variables/mixins available to us already for a bit of this.